datasets/paws-x/paws-x.py

```diff
@@ -0,0 +1,175 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PAWS-X, a multilingual version of PAWS for six languages."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{pawsx2019emnlp,
+  title = {{PAWS-X: A Cross-lingual Adversarial Dataset for Paraphrase Identification}},
+  author = {Yang, Yinfei and Zhang, Yuan and Tar, Chris and Baldridge, Jason},
+  booktitle = {Proc. of EMNLP},
+  year = {2019}
+}
+"""
+
+_DESCRIPTION = """\
+PAWS: Paraphrase Adversaries from Word Scrambling
+
+This dataset contains 23,659 human translated PAWS evaluation pairs and 296,406 machine
+translated training pairs in six typologically distinct languages: French, Spanish, German,
+Chinese, Japanese, and Korean. English language is available by default. All translated
+pairs are sourced from examples in PAWS-Wiki.
+
+For further details, see the accompanying paper: PAWS-X: A Cross-lingual Adversarial Dataset
+for Paraphrase Identification (https://arxiv.org/abs/1908.11828)
+
+NOTE: There might be some missing or wrong labels in the dataset and we have replaced them with -1.
+"""
+
+_HOMEPAGE = "https://github.com/google-research-datasets/paws/tree/master/pawsx"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = 'The dataset may be freely used for any purpose, although acknowledgement of Google LLC ("Google") as the data source would be appreciated. The dataset is provided "AS IS" without any warranty, express or implied. Google disclaims all liability for any damages, direct or indirect, resulting from the use of the dataset.'
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_DATA_OPTIONS = [
+    "en",
+    "de",
+    "es",
+    "fr",
+    "ja",
+    "ko",
+    "zh",
+]
+
+_DATA_URL = "https://storage.googleapis.com/paws/pawsx/x-final.tar.gz"
+
+
+class PAWSXConfig(datasets.BuilderConfig):
+    """BuilderConfig for PAWSX."""
+
+    def __init__(self, **kwargs):
+        """Constructs a PAWSXConfig.
+        Args:
+            **kwargs: keyword arguments forwarded to super.
+        """
+        super(PAWSXConfig, self).__init__(version=datasets.Version("1.1.0", ""), **kwargs),
+
+
+class PAWSX(datasets.GeneratorBasedBuilder):
+    """PAWS-X, a multilingual version of PAWS for six languages."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        PAWSXConfig(
+            name=config_name,
+            description=(f"This config contains samples in {config_name}."),
+        )
+        for config_name in _DATA_OPTIONS
+    ]
+
+    DEFAULT_CONFIG_NAME = "en"  # It's not mandatory to have a default configuration. Just use one if it make sense.
```

Comment:
```suggestion
```
I'm not sure it makes sense to put one language forward compared to the others

Reply:
Sure, I'll make that change. Let the user decide what they want